core-utilities/README.md

```diff
@@ -349,3 +349,8 @@ for (int i = 0; i < ta.size(); i++) {
 ##Citation
 
 Thank you for citing us if you use us in your work! 
+
+```
+@inproceedings{
```

Comment:
If you use this code in your research, please provide the URL for this 
github repository in the relevant publications.